README.md

```diff
@@ -387,6 +387,22 @@ Again, with all these methods, You will want your application to link to an inte
 > NOTE:
 > A JWT will expire after 60 seconds of creation.
 
+#### Fetch Stormpath Access Token with username and password
+Stormpath can generate a brand new Access Token using the password grant type: User's credentials.
+
+To fetch the oauth token use the following snippet
+```ruby
+grant_request = Stormpath::Oauth::PasswordGrantRequest.new(account.email, account.password)
```

Comment:
I'd put `Stormpath::Oauth::PasswordGrantRequest.new(email, password)` since developers will probably not have access to the email and password through an account.